cmd/oras/blob/push.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package blob
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+
+	digest "github.com/opencontainers/go-digest"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"oras.land/oras/cmd/oras/internal/option"
+)
+
+type pushBlobOptions struct {
+	option.Common
+	option.Remote
+
+	FileRef   string
```

Comment:
I noticed we have `FileRefs []string` in Pusher option struct:
```
type Pusher struct {
	ManifestExportPath     string
	PathValidationDisabled bool
	ManifestAnnotations    string

	FileRefs []string
}
```
However, the above three fields are not necessary for the blob push case. And for now we are only considering push a blob, so we don't need `FileRefs` as a slice.

Reply:
`FileRef` doesn't need to be public. It has to be public in `pusher` because the option(a bunch of flags and arguments) will be resued by `oras push` and `oras attach` for packing.

Reply:
Got it, changed to unexported(lowercase), Thanks for pointing out!